backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -87,6 +87,11 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
   /** The type of the run status returned during each poll. */
   type StandardAsyncRunStatus
 
+  /** True if the status contained in `thiz` is equivalent to `that`, delta any other data that might be carried around
+    * in the status type but which doesn't actually represent the status.
+    */
+  def equivalentTo(thiz: StandardAsyncRunStatus)(that: StandardAsyncRunStatus): Boolean = thiz == that
```

Comment:
I think this comment is not accurate for the default implementation - it compares everything

Reply:
Yes you're right, will fix.